### PR TITLE
add welcome message for PR auto replies

### PR DESCRIPTION
**Pull request by colby-swandale**
This PR is adding the welcome message that will serve as the auto reply for Bundlerbot when a new PR is opened.

### CODE REVIEW

File: .github/welcome_message.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Thanks for opening a pull request and helping make Bundler better! Someone from the Bundler team will take a look at your pull request shortly and leave any feedback. Please make sure that your PR has tests for any changed or added functionality.
```
Comment (segiddins) [inline .github/welcome_message.md]: `changes`

**Comment by soberingsally** [comment]
Thanks so much! I'm new to all this so I am still figuring things out lol!
Appreciate the help.

**Comment by colby-swandale** [comment]
@bundlerbot r+

**Comment by indirect** [comment]
🎉

**Pull request merged**